core/src/main/java/tech/tablesaw/io/csv/CsvReadOptions.java

```diff
@@ -283,5 +286,11 @@ public Builder minimizeColumnSizes() {
       super.minimizeColumnSizes();
       return this;
     }
+
+    /** {@inheritDoc} */
```

Comment:
We use `@Override` instead of `{@inheritDoc}`

Reply:
Thanks, I'll fix it.